src/patterns/components/navigation/narrow-navigation.hbs

```diff
@@ -0,0 +1,71 @@
+---
+name: Narrow Navigation
+notes: |
+  This component is intended for narrow viewports. On wide_wide-navigation.scss viewports, you should use the wide navigation component.
```

Comment:
Maybe a search and replace typo. I think wide_wide-navigation.scss needs to be "wide".